keps/sig-auth/4633-anonymous-auth-configurable-endpoints/README.md

```diff
@@ -1008,7 +1073,7 @@ Why should this KEP _not_ be implemented?
 The following should be resolved before this goes to `beta`:
 
 - Should we apply any restrictions here to anonymous `userInfo` that comes back
```

Comment:
@liggitt we need to make a call on fixing this. What are your thoughts? I think while it is related to authentication it is a separate concern from the intent of the KEP itself.

Reply:
A user that arrives authenticated and is allowed to impersonate system:anonymous is not actually anonymous. I think that is out of scope for this KEP.

This KEP is focused on giving cluster admins a tool to constrain actually anonymous requests. I think it is reasonable to expect cluster admins 1) also control their other configured authenticators, and that 2) other authenticators don't authenticate as `system:anonymous`.

Admins can already disallow `system:anonymous` being returned from configured JWT authenticators via `userValidationRules`. If we wanted to add more options to webhook authenticators and add `userValidationRules` capabilities there as well, we could consider that, but that is a separate effort.

I wouldn't fold in changes for those to this KEP.